superset/connectors/druid/models.py

```diff
@@ -107,24 +107,29 @@ def data(self):
             'backend': 'druid',
         }
 
+    @staticmethod
+    def get_base_url(host, port):
+        if not host.startswith('http'):
```

Comment:
It's unlikely, but this might cause problems if the hostname starts with `http`, eg, `http-server.example.com`. I think it's better to use a regex here:

```python
if not re.match('http(s)?://', host):
    host = 'http://' + host
```

Reply:
good call